pkgs/development/lua-modules/overrides.nix

```diff
@@ -711,6 +711,8 @@ in
     };
   });
 
+  lux-lua = final.callPackage ./lux-lua.nix { inherit lua; };
```

Comment:
Ignore last comment, I see in the `installPhase` you're installing it as a lua directory. 

Should we get started on an approach for not crowding `overrides.nix` for package definitions, like we did with `vimPlugins`.

Reply:
I think we can defer the discussion to avoid polluting an important and already commented PR.

Reply:
Well, this is introducing the concept into `overrides.nix`, though.

Reply:
Personally, I would do this in a separate PR.

Reply:
@khaneliman you would prefer it to live in pkgs/top-level/lua-packages.nix ? which might make more sense maybe

Reply:
Yeah, something like that. But, we can handle that later if we want to iron out the details and come up with a general approach